### PR TITLE
chore(issue-templates): updating issue templates for the website

### PR DESCRIPTION
**Pull request by jeffchew**
### Related Ticket(s)

No related issue

### Description

This will update the issue templates to be more focused on the website rather than the Carbon for IBM.com library. This also introduced issue forms.

Live example: https://github.com/jeffchew/carbon-for-ibm-dotcom-website/issues/new/choose

### Changelog

**New**

- bug, feature request, and user story issue forms

**Removed**

- bug, feature request, and question issue markdown templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    attributes:
+      value: 'Use the form below to report any bugs you find on the Carbon for IBM.com website. Feel free to leave blank any sections that aren’t relevant but try to include as much detail as possible. The more details we have, the faster we can respond.'
+  - type: markdown
+    attributes:
+      value: 'If this is an issue for Carbon for IBM.com and not the website, please open an issue at https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/new/choose.'
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: "What did you expect to happen? What happened instead?"
```
Comment (shixiedesign) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:22]: I suggest we add "Please attach a screenshot if relevant."

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    id: problem
+    attributes:
+      label: The problem
+      description: "A clear and concise description of what the problem this new feature is trying to solve. (ex. I'm always frustrated when...)"
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: The solution
```
Comment (shixiedesign) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:25]: How about "Proposed solution"?

**Review by shixiedesign — COMMENTED** [review]
For "Feature requests" in website world, I'd hope people can provide sample content, references, or audits of solutions solving the same problem in other companies. Potentially more applicable than "user insight / research / metric", which I imagine people might gloss over.

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (jeffchew) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:22]: @shixiedesign we have the question below asking for screenshots, should we move to the first question instead?

**Comment by jeffchew** [comment]
> For "Feature requests" in website world, I'd hope people can provide sample content, references, or audits of solutions solving the same problem in other companies. Potentially more applicable than "user insight / research / metric", which I imagine people might gloss over.

Updated! I consolidated the question to be more direct with asking for sample content/references: https://github.com/jeffchew/carbon-for-ibm-dotcom-website/issues/new?assignees=jeffchew%2Cljcarot%2Cshixiedesign%2CRobertaJHahn&labels=Feature+request&template=feature_request.yaml&title=%5BYOUR+TITLE%5D%3A+Brief+description

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
+name: Bug report 🐛
+description:
+  Something isn't working as expected? Here is the right place to report.
```
Comment (janchild) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:3]:
```suggestion
  Something isn't working as expected? Here's the right place to report.
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    attributes:
+      label: The problem
+      description: "A clear and concise description of what the problem this new feature is trying to solve. (ex. I'm always frustrated when...)"
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: The proposed solution
+      description: "A clear and concise description of what you want to happen"
```
Comment (janchild) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:26]:
```suggestion
      description: "Provide a clear and concise description of what you would like to happen instead."
```

File: .github/ISSUE_TEMPLATE/user_story.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+      - Each product backlog item or user story should have at least one Acceptance criteria.
+
+      - Acceptance criteria defines a deliverable that can be completed in a single sprint.
+
+      - Each Acceptance criterion is independently testable.
+
+      - Include functional as well as non-functional criteria – when relevant.
+
+      - Team members write Acceptance criteria and the Product Owner verifies it."
```
Comment (janchild) [inline .github/ISSUE_TEMPLATE/user_story.yaml:21]:
```suggestion
      value: "Consider the following when writing acceptance criteria for this story:

      - Each product backlog item or user story should have at least one Acceptance criteria.

      - Acceptance criteria defines a deliverable that can be completed in a single sprint.

      - Each acceptance criterion is independently testable.

      - Include functional as well as non-functional criteria – when relevant.

      - Team members write acceptance criteria and the product owner verifies it."
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+      description: Please confirm the following
+      options:
+        - label:
+            I agree to follow this project's [Code of
+            Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/blob/master/.github/CODE_OF_CONDUCT.md)
+          required: true
+        - label:
+            I checked the [current
+            issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/issues) for
+            duplicate issues
```
Comment (janchild) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:46]:
```suggestion
            I checked the [current
            issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/issues) for
            duplicate issues.
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: user-research
+    attributes:
+      label: Sample content, references, or audits of solutions solving the same problem in other websites
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
```
Comment (janchild) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:36]:
```suggestion
      label: Code of conduct
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+    id: solution
+    attributes:
+      label: The proposed solution
+      description: "A clear and concise description of what you want to happen"
+    validations:
+      required: true
+  - type: textarea
+    id: user-research
+    attributes:
+      label: Sample content, references, or audits of solutions solving the same problem in other websites
```
Comment (janchild) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:32]:
```suggestion
      label: Examples
      description: Provide sample content, references, or audits of solutions solving the same problem in other websites.
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+      value: '## Feature Request 💡'
+  - type: markdown
+    attributes:
+      value: 'If this is a feature request for Carbon for IBM.com and not the website, please open an issue at https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/new/choose.'
+  - type: textarea
+    id: problem
+    attributes:
+      label: The problem
+      description: "A clear and concise description of what the problem this new feature is trying to solve. (ex. I'm always frustrated when...)"
+    validations:
```
Comment (janchild) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:20]:
```suggestion
      description: "Provide a clear and concise description of what the problem this new feature is trying to solve. (For example, I find it frustrating when...)"
    validations:
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+assignees:
+  - jeffchew,ljcarot,shixiedesign,RobertaJHahn
+labels: [ 'Feature request' ]
+body:
+  - type: markdown
+    attributes:
+      value: '## Feature Request 💡'
+  - type: markdown
+    attributes:
+      value: 'If this is a feature request for Carbon for IBM.com and not the website, please open an issue at https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/new/choose.'
```
Comment (janchild) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:14]:
```suggestion
      value: 'If this is a feature request for Carbon for IBM.com and not the Carbon for IBM.com website, please open an issue at https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/new/choose.'
```

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+      description: Please confirm the following
+      options:
+        - label:
+            I agree to follow this project's [Code of
+            Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/blob/master/.github/CODE_OF_CONDUCT.md)
+          required: true
+        - label:
+            I checked the [current
+            issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/issues) for
+            duplicate issues
```
Comment (janchild) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:60]:
```suggestion
            I checked the [current
            issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/issues) for
            duplicate issues.
```

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+      label: "Steps to reproduce the issue (if applicable)"
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: Please confirm the following
+      options:
+        - label:
+            I agree to follow this project's [Code of
+            Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/blob/master/.github/CODE_OF_CONDUCT.md)
```
Comment (janchild) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:55]:
```suggestion
            I agree to follow this project's [Code of
            Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/blob/master/.github/CODE_OF_CONDUCT.md).
```

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (janchild) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:22]: I took a look and with the layout it works to keep the screenshot where you have it under Pages impacted. I like the division of topics with these headings. It looks easy to complete and inviting!

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 13]
```diff
@@ -0,0 +1,60 @@
...
+        - Firefox
+        - Microsoft Edge
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: "Steps to reproduce the issue (if applicable)"
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
```
Comment (janchild) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:50]:
```suggestion
      label: Code of conduct
```

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 12 above)
Comment (janchild) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:55]:
```suggestion
            conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/blob/master/.github/CODE_OF_CONDUCT.md)
```

**Review by janchild — APPROVED** [review]
I like these! Just a few content tweaks that I think will help but I'm happy to approve.

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+      label: "Steps to reproduce the issue (if applicable)"
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of conduct
+      description: Please confirm the following
+      options:
+        - label:
+            I agree to follow this project's [Code of
+            conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/blob/master/.github/CODE_OF_CONDUCT.md)
```
Comment (jeffchew) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:55]:
```suggestion
            conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/blob/master/.github/CODE_OF_CONDUCT.md).
```

**Comment by jeffchew** [comment]
Thank you for reviewing @janchild and @shixiedesign !

**Pull request merged**